Hash_map/include/CGAL/Tools/chained_map.h

```diff
@@ -37,42 +37,33 @@ class chained_map_elem
 template <typename T, typename Allocator>
 class chained_map
 {
-   const std::size_t nullptrKEY;
-   const std::size_t NONnullptrKEY;
-
-   chained_map_elem<T> STOP;
+   static constexpr std::size_t nullptrKEY = (std::numeric_limits<std::size_t>::max)();
```

Comment:
I think you should modify that variable name. It was a victim of the transformation `NULL`->`nullptr`.

It should be renamed `NULLKEY` (its original name), or `nullkey` (that is better because it satisfies our naming guildelines).